lib/src/authentication_service.dart

```diff
@@ -105,14 +108,15 @@ class AuthenticationService {
       _updateStatus(AuthenticationStatus.authenticating);
       await pca!.acquireToken(defaultScopes);
       _updateStatus(AuthenticationStatus.authenticated);
-    } catch (e) {
+    } on MsalException catch (e) {
```

Comment:
keep this catching any exception. Regardless of the exception type it should have failed.